confluence-xml/src/test/resources/confluencexml/alert.test

```diff
@@ -44,7 +44,8 @@
               <string>content</string>
               <string>== {{id name="MySpaceHome-alert"/}}alert ==
 
-{{success title="Wohoo! u succeeded" type="Success"}}
+{{success title="Wohoo! u succeeded"}}
```

Comment:
Again, since you handled the parameters, there is no need for the xwiki macro to have "title" and "type" parameter. And the xwiki message macros do not support these parameters.